features/admin.server-configurations.v1/forms/password-recovery-form/password-recovery-form.tsx

```diff
@@ -0,0 +1,802 @@
+                        enableSMSBasedRecovery: CommonUtils.parseBoolean(property.value)
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_SMS_EXPIRY_TIME) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        smsOtpExpiryTime: property.value
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_OTP_USE_UPPERCASE) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        passwordRecoveryOtpUseUppercase: CommonUtils.parseBoolean(property.value)
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_OTP_USE_LOWERCASE) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        passwordRecoveryOtpUseLowercase: CommonUtils.parseBoolean(property.value)
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_OTP_USE_NUMERIC) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        passwordRecoveryOtpUseNumeric: CommonUtils.parseBoolean(property.value)
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_OTP_LENGTH) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        smsOtpLength: property.value
+                    };
+                } else if (property.name === ServerConfigurationsConstants.RECOVERY_MAX_RESEND_COUNT) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        maxResendCount : property.value
+                    };
+                }  else if (property.name === ServerConfigurationsConstants.RECOVERY_MAX_FAILED_ATTEMPTS_COUNT) {
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        maxFailedAttemptCount : property.value
+                    };
+                }
+            }
+        });
+        setInitialConnectorValues(resolvedInitialValues);
+        setIsEmailRecoveryEnabled(resolvedInitialValues?.enableEmailBasedRecovery);
+        setIsSMSRecoveryEnabled(resolvedInitialValues?.enableSMSBasedRecovery);
+        setIsUpperCaseEnabled(resolvedInitialValues?.passwordRecoveryOtpUseUppercase);
+        setIsLowerCaseEnabled(resolvedInitialValues?.passwordRecoveryOtpUseLowercase);
+        setIsNumericEnabled(resolvedInitialValues?.passwordRecoveryOtpUseNumeric);
+    }, [ initialValues ]);
+
+    /**
+     * Validate input data.
+     *
+     * @param values - Form values.
+     * @returns Form validation.
+     */
+    const validateForm = (values: any):
```

Comment:
refrain from using `any`

Reply:
this is also previous implementation detected as new due to file move.